sklego/ensemble_model.py

```diff
@@ -0,0 +1,87 @@
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix
+
+class EnsembleModel():
+    """
+    A classifier that takes the uncertainty p(class | model) of a classifier into account.
+    Based on the requirements of issue #23 of koaning/scikit-lego
+    """
+
+    def __init__(self, X_test, y_test, estimator=LogisticRegression(), alpha=0.5):
+        """
+
+        :param X_test: array of feature values that will be used to test the classifier and create a confusion matrix
+        :param y_test: array of target values that will be used to test the classifier and create a confusion matrix
+        :param estimator: sklean classifier;
+                          default one is LogisticRegression, but others can be used too (eg. Decision Tree)
+        :param alpha: hyperparameter; needs to be optimized to increase classification accuracy
+        """
+        self.estimator = estimator
+        self.alpha = alpha
+        self.X_test = X_test
+        self.y_test = y_test
+
+    def fit(self, X, y):
+        """
+
+        :param X: array of feature values used for training the classifier
+        :param y: array of target values used for training the classifier
+        :return: trained classifier (estimator)
+        """
+        self.fitted_estimator_ = self.estimator.fit(X, y)
```

Comment:
perhaps a different name than `self.fitted_estimator_` maybe ... `self.orig_estimator` 

also you may want to calculate the confusion matrix in the `self.fit` as well